### DISCUSSION TITLE
Ore Generators in the Islands

### DISCUSSION THREAD

**Discussion by JLucassz** [BG-Software-LLC/SuperiorSkyblock2#242] [Denied Feature Requests] [answered]
Add an option in the config.yml for the ores on the islands to appear according to the island level: for example, coal and iron appear in the generator when the island is at level 80, gold appears at level 150. So the island level is it would make it more useful and the player would unlock it, I would love to use it on my server. I hope it can be added. It would look something like this:
![ilha geradores](https://user-images.githubusercontent.com/76066285/117473326-36f13300-af30-11eb-9ead-cdd9130e6137.png)

**Comment by OmerBenGera** [comment, marked as answer]
You can already achieve this using the missions system.
